src/main/java/org/folio/mr/service/impl/MediatedRequestActionsServiceImpl.java

```diff
@@ -38,6 +44,11 @@ public MediatedRequest confirmItemArrival(String itemBarcode) {
     return dto;
   }
 
+  @Override
+  public MediatedRequestWorkflowLog saveMediatedRequestWorkflowLog(MediatedRequest request) {
+    return workflowLogRepository.save(buildLogByRequest(request));
```

Comment:
Add logging to this method (no need to log the whole request here, just something like "Creating mediated request workflow log entry for request {ID}")